internal/cmd/deployrequest/deploy.go

```diff
@@ -64,13 +64,23 @@ func DeployCmd(ch *cmdutil.Helper) *cobra.Command {
 					Database:     database,
 					Number:       n,
 				}
-				if err := waitUntilReady(ctx, client, ch.Printer, ch.Debug(), getReq); err != nil {
+				state, err := waitUntilReady(ctx, client, ch.Printer, ch.Debug(), getReq)
+				if err != nil {
 					return err
 				}
 				end()
 
-				ch.Printer.Printf("Deploy request %s/%s is successfully deployed.",
-					printer.BoldBlue(database), printer.BoldBlue(number))
+				switch state {
+				case "complete_pending_revert":
+					ch.Printer.Printf("Deploy request %s/%s is successfully deployed and revertable. You can skip the revert to unblock the deploy queue.\n",
+						printer.BoldBlue(database), printer.BoldBlue(number))
+				case "pending_cutover":
+					ch.Printer.Printf("Deploy request %s/%s is successfully staged and waiting to be applied.\n",
+						printer.BoldBlue(database), printer.BoldBlue(number))
+				default:
```

Comment:
What is the default value? We'll have to include that also in `waitUntilReady()`, otherwise it won't succeed and spin forever until we hit the 5 minute timeout.

Reply:
included the default state `complete`